src/01-functions.js

```diff
@@ -168,7 +194,9 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+  return a.slice(0,1) + a.slice(-1)
```

Comment:
very nice!  Remember - we can index strings like arrays.  Can we think of another way to do this?